prow/jobs/images/Dockerfile.build-prow-images

```diff
@@ -14,14 +14,40 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ack-build-tools ./prow/job
 
 FROM busybox:musl AS busybox
```

Comment:
Are we still using busybox?

Reply:
no, that was left from previous one

Reply:
Looks like you still have it on your commit https://github.com/michaelhtm/ack-test-infra/blob/fix-build-prow-images/prow/jobs/images/Dockerfile.build-prow-images#L15